### PR TITLE
[kube-prometheus-stack] Update values.yaml

### PR DESCRIPTION
**Pull request by xogoodnow**
#### What this PR does / why we need it

There is no mention on how to persist Grafana in this file. I believe it would be more straight forward to add it here, comparing to reading the charts in order to understand the "how to"
So I propose this section as comment for better usability.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/values.yaml [hunk 1]
```diff
@@ -980,6 +980,18 @@ grafana:
     #   hosts:
     #   - grafana.example.com
 
+  # # To make Grafana persistant (Using Statefulset)
```
Comment (QuentinBisson) [inline charts/kube-prometheus-stack/values.yaml:983]:
```suggestion
  # # To make Grafana persistent (Using Statefulset)
```

File: charts/kube-prometheus-stack/values.yaml [hunk 2]
```diff
@@ -980,6 +980,18 @@ grafana:
...
+  # #
+  # persistence:
+  #   enabled: true
+  #   type: sts
+  #   storageClassName: <StorageClassName>
```
Comment (QuentinBisson) [inline charts/kube-prometheus-stack/values.yaml:988]:
```suggestion
  #   storageClassName: 'storageClassName"
```

**Comment by jkroepke** [comment]
@xogoodnow do you want to apply the suggestions? Please also bump the version inside Chart.yaml. Thanks.

**Comment by xogoodnow** [comment]
Hi @jkroepke, 
Sure, applied the changes.
Cheers

**Comment by jkroepke** [comment]
It seems like you bump the version first, then merge. But there is no version increment.

**Comment by xogoodnow** [comment]
Hi @jkroepke,
My bad, I incremented the chart version
Cheers

**Review by jkroepke — APPROVED** [review]
Thanks!

**Pull request merged**